release-notes.txt

```diff
@@ -1 +1,30 @@
-Initial app version.
\ No newline at end of file
+---- Thursday 27th March, 2020 ----
```

Comment:
@maks and @workerbee22 I'm not entirely sure about this format?  I'm hoping it will work out well enough in firebase app dist?

Feel free to suggest changes to this as you see fit

Reply:
@lukesleeman  I'd add the need to update the release notes on both App Store and Google Play listings as I dont know if there is any automation available in Codemagic to upload that as part of process of distributing to stores, though @workerbee22 may know of a script to do that.